modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,86 @@
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+* Organization Administrators permissions to access all the business groups and environments in which your API instances are located. 
++
+Otherwise, you may not find the required organization or business group in the list.
+
+[[create-api-groups]]
+== Create API Groups
+
+You create an API Group at the organization level.
+
+To create API Groups:
+
+. From the left menu in API Manager, click *API Groups > Create API Groups*.
++
+The Create New API Group window appears.
+. In the *API Group Name* field, specify a name for the API Group that you want to create: for example, `Social Media`.
+. In the *API Group version* field, specify the version for the API Group, for example, `v1`.
+. Optionally, in the API Group instance label field, specify a label for the API Group, such as `Internal`, if it is to be used for internal purposes.
+. Click *Ok*.
++
+The *Creating a new group* page is displayed. You are now ready to <<adding-api-instances-to-groups,add API instances>> to the API Group.
+
+
+[[adding-api-instances-to-groups]]
+== Add API instances to API Groups
+
+Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. You can add API instances that belong in the same business group or organizations.
+
+Note that you can add only those API instances that belong the same environment as the API Group instance.
+
+To add a new API instance to a new API Group:
+
+. On the *Creating a new group* page, select an environment from the *Select environment* drop-down list.
++
+.The screenshot illustrates the fields and options displayed on the Creating a new group page.
+image:create-api-instances-api-groups.png[Adding API instances to API Groups]
++
+<1> The organization under which the API Group is created.
+<2> The Add button to add more API instances to the group.
+<3> The Add Organization button to add organizations to the group.
+<4> The Save button.
+<5> The delete button to instantly delete an API instance from the group.
+. From the *Select an API* list, choose the API instance that you want to add.
+. From the *Select a version / instance* list, choose the version of the API Instance that you want to add to the Group.
+. If you want to include additional API instances in the API Group, click *Add* and specify the details.
+. If the API instance and the API Group belong in different business groups:
+.. Click *Add Organization*.
+.. Select the organization or business group in which your API instance is available.
+.. From the environment, select the API instance, and version that you want to add.
+. Click *Save*.
++
+Your API instances are now added to the API Group.
```

Comment:
We should also add another page about the promotion flow

Reply:
Done! Thanks!